files/es/mdn/structures/macros/other/index.html

```diff
@@ -85,19 +85,15 @@ <h3 id="Transclusión">Transclusión</h3>
 
 <h4 id="Ejemplo_sin_título">Ejemplo sin título</h4>
 
-<p>\{{Page("/es/docs/MDN/About", "Acerca de Mozilla")}}</p>
```

Comment:
I'm not quite sure if the occurrences in this file should be deleted. The en-US docs preserved them as a demonstration of how to use this macro (despite it being deprecated), which I didn't notice when I was searching for the macro across all files

Reply:
good to know, thank you!